pandas/tests/window/moments/test_moments_consistency_rolling.py

```diff
@@ -136,6 +136,54 @@ def test_rolling_apply_consistency(
                 tm.assert_equal(rolling_f_result, rolling_apply_f_result)
 
 
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    "window,min_periods,center", list(_rolling_consistency_cases())
+)
+def test_rolling_groupby(base_functions, window, min_periods, center):
+    base_df = DataFrame({"group": "A", "data": randn(20)})
+
+    b_df = base_df.copy()
+    b_df["group"] = "B"
+
+    grp_df = pd.concat([base_df, b_df]).groupby("group")
+
+    for (f, require_min_periods, name) in base_functions:
+        if (
+            require_min_periods
+            and (min_periods is not None)
```

Comment:
why are these skipped?

Reply:
if min_periods is less than the required_min_periods then there will be an error thrown so we wouldn't be able to test the equivalency of the two dfs

Reply:
If there's an error thrown test for that using `pytest.raises`